mix.exs

```diff
@@ -1,12 +1,12 @@
 defmodule ScrivenerHtml.Mixfile do
   use Mix.Project
 
-  @version "1.9.0"
+  @version "2.0.0"
   def project do
     [
       app: :scrivener_html,
       version: @version,
-      elixir: "~> 1.14",
+      elixir: "~> 1.16.3",
```

Comment:
```suggestion
      elixir: "~> 1.16",
```

Please relax this, what you're using means that you can only use 1.16.x, and not 1.17 or later.